src/app/map/item-location.service.ts

```diff
@@ -25,15 +31,83 @@ export class ItemLocationService {
       this.isKingsTombAvailable,
       this
     ));
+    this.itemLocations.set( LocationKey.LightWorldSwamp, new ItemLocation(
+      'Light World Swamp',
+      '',
+      45,
+      91,
+      ItemLocationService.always,
+      this
+    ));
+    this.itemLocations.set( LocationKey.LinksHouse, new ItemLocation(
+      'Link\'s House',
+      '',
+      52.5,
+      65.3,
+      ItemLocationService.always,
+      this,
+      _settings.mode === Mode.Standard
+    ));
+    this.itemLocations.set( LocationKey.SpiralCave, new ItemLocation(
+      'Spiral Cave',
+      'East Death Mountain Access',
+      77.9,
+      6.6,
+      this.isSpiralCaveAccessible,
+      this
```

Comment:
Stuff like lines 56 and 57 are my concern. I have a class dependent on a service for its data. Is there a better approach?